src/bloxone_client/api_client.py

```diff
@@ -184,7 +190,17 @@ def param_serialize(self,
 
         # header parameters
         header_params = header_params or {}
-        header_params.update(self.default_headers)
+        headers = {
+            header_client: self.configuration.client_name,
+            header_sdk: sdk_identifier,
+            'Authorization': 'f"Token {self.configuration.api_key}"',
+            "Content-Type": "application/json",
+            "Accept": "application/json",
```

Comment:
Are these necessary? They should be set else where. For e.g. 

https://github.com/infobloxopen/bloxone-python-client/blob/a0c9c9e449012007a54aba9df183a6866be7883e/src/bloxone_client/api_client.py#L534-L543

Reply:
Added these 3 headers to `update_params_for_auth`